src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -422,6 +422,54 @@ public function configureFilesystemAdapter(ContainerBuilder $container, array $c
             $container->removeDefinition('sonata.media.filesystem.s3');
         }
 
+        // add the default configuration for the Spaces filesystem
+        if ($container->hasDefinition('sonata.media.adapter.filesystem.spaces') && isset($config['filesystem']['spaces'])) {
+            $container->getDefinition('sonata.media.adapter.filesystem.spaces')
+                ->replaceArgument(0, new Reference('sonata.media.adapter.service.spaces'))
+                ->replaceArgument(1, $config['filesystem']['spaces']['bucket'])
+                ->replaceArgument(2, ['create' => $config['filesystem']['spaces']['create'], 'region' => $config['filesystem']['spaces']['region'], 'directory' => $config['filesystem']['spaces']['directory'], 'ACL' => $config['filesystem']['spaces']['acl']])
+            ;
+
+            $container->getDefinition('sonata.media.metadata.amazon')
```

Comment:
amazon?

Reply:
from what i understand DO Spaces is s3-compliant. However, that is a good question if we need it at all. It may be required by Guafrette.